docs/syntax-reference.rst

```diff
@@ -39,6 +39,32 @@ means new facts can be inserted to the table, and existing facts can be updated
 Kafka topic or derived from existing streams and tables. In both cases, a table’s underlying data is durably stored (persisted)
 within a Kafka topic on the Kafka brokers.
 
+.. _struct_overview:
+
+STRUCT
+------
+
+In KSQL 5.0 and higher, you can read nested data, in Avro and JSON formats,
+by using the STRUCT type in CREATE STREAM and CREATE TABLE statements. Use the
```

Comment:
To clarify:  IIRC you can use the STRUCT type in:

- CREATE STREAM/TABLE (from a topic)
- CREATE STREAM/TABLE AS SELECT (from existing streams/tables)
- SELECT (= non-persistent query)

I'll let others chime in to confirm.